franklin/github/views.py

```diff
@@ -255,8 +257,11 @@ def get_access_token(request):
     if status.is_success(r.status_code):
         try:
             access_token = r.json().get('access_token', None)
+            user = do_auth(access_token)
+            serializer = UserSerializer(user)
             response_data = Response({
-                'token': access_token
+                'token': access_token,
+                'user': serializer.data
```

Comment:
with `do_auth` occurring here, the user will now exist in our DB during the step when we convert the temporary token for a github oauth token. We then serializer the user model (which just returns username for now) and return that json object as part of the response to the dashboard.